app/views/dashboard/_index_partials/_stats.html.erb

```diff
@@ -4,12 +4,16 @@
 
   <table class="table table-condensed stats">
     <tr>
-      <td>You have deposited <span><%= number_of_files %> files</span> which have been:
+      <td>You have deposited <span><%= number_of_files %> files</span> 
+<!-- This is temporarily removed because of issue #889. And it will be fixed and restored in Sufia 7 -->
+<!--
+which have been:
 
         <ul class="views-downloads-dashboard">
           <li>- viewed <span><%= @user.total_file_views %></span> times</li>
           <li>- downloaded <span><%= @user.total_file_downloads %></span> times</li>
```

Comment:
Can we delete this? Or comment it out via Ruby ```<%# ... %> ``` tags? That way these call don't get run and the results rendered in the comments